lib/improver/utilities/cube_checker.py

```diff
@@ -144,48 +143,42 @@ def check_cube_coordinates(cube, new_cube, exception_coordinates=None):
     for coord in new_cube.aux_coords[::-1]:
         if coord.name() in cube_dim_names:
             new_cube = iris.util.new_axis(new_cube, coord)
+    new_cube_dim_names = [coord.name() for coord in new_cube.dim_coords]
```

Comment:
You are setting this variable but don't seem to be using it.

Reply:
The only place I am using it is in the error message below. I can move it to just before the error message if you think that's clearer

Reply:
Sorry, I must of missed that Doh!. Leave as it is, it is fine.